sdk/graphrbac/graph/README.md

```diff
@@ -68,7 +71,7 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
     <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
     <script src="node_modules/@azure/graph/dist/graph.js"></script>
     <script type="text/javascript">
-      const subscriptionId = "<Subscription_Id>";
+      const tenantId = "<Tenant_Id>";
```

Comment:
How is this `tenantId` different from the one in line 33?

Reply:
Hi @ramya-rao-a - they are part of separate code examples. The tenantId on line 74 is part of the browser-specific example, whereas the one on line 33 you mentioned is part of the Node.js example.

Reply:
Oh, thats right. Apologies, that was an oversight from my end :)

Reply:
In that case, I would suggest to update line 33 to `const tenantId = "<Tenant_Id>";` as well because we dont talk about the environment variable DOMAIN needing to be set anywhere in the readme.

Reply:
Sure, that makes sense. I was trying to change the original code as little as possible and since it referenced an env var for subscription id prior to my other updates. 

There's some additional code that can be removed (the following imports on lines 29-30 aren't used and aren't necessary for the example to run, I have tested this):
```
import * as msRest from "@azure/ms-rest-js";
import * as msRestAzure from "@azure/ms-rest-azure-js";
```

I will add a commit with these changes shortly.